frame/clearing-house/src/tests/update_funding.rs

```diff
@@ -35,6 +44,34 @@ prop_compose! {
 	}
 }
 
+prop_compose! {
+	fn balance_in_open_interval(lower_bound: Balance, upper_bound: Balance)(
+		balance in (lower_bound + 1)..upper_bound
+	) -> Balance {
+		balance
+	}
+}
+
+prop_compose! {
+	fn long_short_amounts_for_capped_funding()(
+		alice_position in any_balance()
+	)(
+		alice_position in Just(alice_position),
+		net_position in balance_in_open_interval(alice_position / 3, alice_position),
+	) -> (Balance, Balance) {
+		// alice_position = bob_position + net_position
+		// fees = (alice_position + bob_position) * 1%
+		//      = 2 * bob_position * 1% + net_position * 1%
+		// bob_funding = (alice_position - net_position) * 5%
+		// alice_funding = alice_position * 5%
+		//               > 2 * (alice_position - net_position) * 1% + net_position * 1%
+		//                 + (alice_position - net_position) * 5%
+		//               = alice_position * 7% - net_position * 6%
+		// <=> net_position > alice_position / 3
```

Comment:
Maybe add a `TODO(0xangelo): ...` here to take care of this commented section?

Reply:
This is actually a 'derivation' of the fact that `net_position` has to be greater than `alice_position / 3` (with these fees and funding rates) for the resulting funding payments to be capped. I should make that more clear

Reply:
Oh I see... :+1: